.ci/docker/requirements-ci.txt

```diff
@@ -85,10 +85,10 @@ librosa>=0.6.2 ; python_version < "3.11"
 #Pinned versions:
 #test that import:
 
-mypy==1.8.0
+mypy==1.9.0
 # Pin MyPy version because new errors are likely to appear with each release
 #Description: linter
-#Pinned versions: 1.8.0
+#Pinned versions: 1.9.0
```

Comment:
You need to update the lintrunner.toml version too.

Reply:
Then it's even more mysterious why it's causing any problems...

Reply:
Actually, we used to have a plugin that prints a warning about mypy discepancy as well as test, but looks like we've forgot about it. Which also makes me wonder: why the heck we still have mypy in `requirements-ci.txt`